### DISCUSSION TITLE
consider to add my profile

### DISCUSSION THREAD

**Discussion by alguerocode** [abhisheknaiidu/awesome-github-profile-readme#610] [Show and tell]
Hi there,
I am alhashmicode, would like to request you to visit my GitHub profile and add it in your repo😊❤ [https://github.com/alhashmicode/alhashmicode ]

Thanking you

**Comment by g0621** [comment]
Consider adding this one too 🙈 
[https://github.com/g0621/g0621](https://github.com/g0621/g0621)

**Comment by TheSmallPixel** [comment]
Add me too, guys!

https://github.com/TheSmallPixel

**Comment by morgandusty** [comment]
Add me too, please)

https://github.com/morgandusty
